frontend/src/app/main/ui/workspace/sidebar/assets.cljs

```diff
@@ -715,21 +756,24 @@
                             :color color
                             :file-id file-id
                             :local? local?
+                            :selected selected
+                            :on-select on-select
                             :locale locale}]))])]))
 
 
 ;; ---- Typography box ----
 
 (mf/defc typography-box
-  [{:keys [file file-id local? typographies locale open?] :as props}]
+  [{:keys [file file-id local? typographies locale open? change-selected] :as props}]
 
   (let [state (mf/use-state {:detail-open? false
                              :menu-open? false
                              :top nil
-                             :left nil})
+                             :left nil
+                             :selected empty-selection})
 
-        selected (mf/deref refs/selected-shapes)
-        local (deref refs/workspace-local)
+        local    (deref refs/workspace-local)
```

Comment:
You're using normal `deref` instead of `mf/use-ref`

Reply:
This was already so before this merge.

I think this is because I do not want the component to repaint every time anything changes in :workspace-local state. Perhaps should be good idea to make a more specific lens.

Reply:
true, this is explicit usage of normal `deref`